pallets/services/src/tests.rs

```diff
@@ -32,6 +32,36 @@ fn zero_key() -> ecdsa::Public {
 	ecdsa::Public([0; 33])
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+enum MachineKind {
```

Comment:
How can we smooth this out? Won't there be variation across what operators consider small, medium, large?

Reply:
Shouldn't there just be a function for pricing cpu, mem, etc. and then we sum the cost chosen? The product can deal with small/medium/large but the system might be better off with more flexibility.

Reply:
This just for testing, I wanted to simulate different machine sizes with each one having different pricing.